app/uk/gov/hmrc/ngrnotifyproto/model/response/ActionCallback.scala

```diff
@@ -30,4 +30,4 @@ case class ActionCallback(
 )
 
 object ActionCallback:
-  implicit val format: OFormat[ActionCallback] = Json.format
+  implicit val format: OFormat[ActionCallback] = Json.format[ActionCallback]
```

Comment:
Looks like incorrect merge result.
We don't need `Json.format[ActionCallback]`, only `Json.format`